builtin/game/item.lua

```diff
@@ -139,6 +139,19 @@ function core.dir_to_wallmounted(dir)
 	end
 end
 
+function core.wallmounted_to_dir(wallmounted)
+	-- table of dirs in wallmounted order
+	return ({[0]={x=0, y=1, z=0},
+					{x=1, y=-1, z=0},
```

Comment:
Ouch.. you are totally right, it was a typo when typing the table from my mod to here, sorry